.rubocop_disabled.yml

```diff
@@ -96,3 +96,6 @@ Rails/UniqueValidationWithoutIndex:
 Rails/UniqueValidationWithoutIndex:
   Enabled: false
 
+Style/HashSyntax:
+  Enabled: false
```

Comment:
sao lại tự ý thêm cái này vào e

Reply:
@manhnt-0960  Nãy em chưa fix được lỗi Style/HashSyntax: Omit the hash value nên em để vào. Bây giờ em xóa rồi ạ